.github/workflows/release.yml

```diff
@@ -0,0 +1,145 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  draft-release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.18
+      - name: Install Carvel Tools
+        uses: vmware-tanzu/carvel-setup-action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
why is token needed here?

Reply:
From: https://github.com/vmware-tanzu/carvel-setup-action#usage
```
carvel-setup-action uses the GitHub API to find information about latest releases. To avoid rate limits it is recommended you pass a token:
```

Reply:
i see, ok. we ll try to resolve this eventually via carvel-release-scripts.